src/app/content/components/TableOfContents/styled/index.tsx

```diff
@@ -128,7 +132,7 @@ export const NavOl = styled.ol<{section: ArchiveTree}>`
       & > ${NavItem} > ${ContentLink} {
         .os-number {
           width: ${numberWidth}rem;
-          overflow: hidden;
+          overflow: visible;
```

Comment:
this is worrying to me, why is it updated?

Reply:
This happens otherwise: https://share.getcloudapp.com/wbu62gY4
I can leave as is and we can discuss since there is another visual issue (the dropdown arrows).

Reply:
there is some hideous math in the sidebar logic that computes the column size based on the text value of the number, that logic is probably wrong now with the new font size